src/Wallabag/CoreBundle/Entity/Entry.php

```diff
@@ -559,6 +559,19 @@ public function setCreatedAt(\DateTimeInterface $createdAt)
         return $this;
     }
 
+    /**
+     * Set updated_at.
+     * Only used for tests.
+     *
+     * @return Entry
+     */
+    public function setUpdatedAt(\DateTimeInterface $updatedAt)
```

Comment:
Don't know if there is a better way to handle this than creating a dedicated setter, maybe using Reflection?

lgtm otherwise

Reply:
I've updated to PR to remove this method and use reflection in the test